plugins/azure-sites/api-report.md

```diff
@@ -23,8 +23,8 @@ export const azureSiteApiRef: ApiRef<AzureSitesApi>;
 // @public (undocumented)
 export type AzureSitesApi = {
   list: (request: AzureSiteListRequest) => Promise<AzureSiteListResponse>;
-  start: (request: AzureSiteStartStopRequest) => Promise<void>;
-  stop: (request: AzureSiteStartStopRequest) => Promise<void>;
+  start: (request: AzureSiteBackendRequest) => Promise<void>;
```

Comment:
I think a better type to use here would be: `AzureSiteStartStopRequest & { entityRef?: string }` rather than creating another type for it?

Reply:
Since this `AzureSiteBackendRequest` is used in many places so decided to keep it in common types instead of adding `AzureSiteStartStopRequest & { entityRef?: string }` in all places.

More over this `AzureSiteStartStopRequest` is used in actual Azure client stop and stop function. So added new type `AzureSiteBackendRequest` for communicate between our frontend & backend.

https://github.com/deepan10/backstage/blob/28b7fe75f91cdf1a2bf19cd4bb552701f4b068bd/plugins/azure-sites-backend/src/api/AzureSitesApi.ts#L58-L68
WDYT?